cloudbridge/providers/azure/azure_client.py

```diff
@@ -310,8 +311,9 @@ def _get_or_create_storage_account(self):
             try:
                 self._storage_account = \
                     self.get_storage_account(self.storage_account)
-            except CloudError as cloud_error:
-                if cloud_error.error.error == "ResourceNotFound":
+            except HttpResponseError as cloud_error:
```

Comment:
I suspect we might need to change all of the `CloudError` to `HttpResponseError` since this is what Azure seems to raise now

Reply:
Makes sense. Maybe we can directly catch `ResourceNotFoundError`?